src/app/learnocaml_index_main.ml

```diff
@@ -848,6 +848,17 @@ let () =
     no_tab_selected ();
     Lwt.return ()
   in
+  let download_all () =
+    let token = get_stored_token ()
+    and name = "archive.tar" in
+    Server_caller.request (Learnocaml_api.Archive_tar token) >|= function
+      | Ok tar ->
+          let contents = Js.string tar in
+          Learnocaml_common.fake_download ~name ~contents
+      | Error e ->
+        alert ~title:"DOWNLOAD ARCHIVE FAILED"
```

Comment:
Please use "internationalized strings" for the title.

Reply:
ok, I forgot to do that